api/scripts/certification/batch-update-certification-frameworks-challenges-from-csv.js

```diff
@@ -0,0 +1,117 @@
+import Joi from 'joi';
+
+import { knex } from '../../db/knex-database-connection.js';
+import { csvFileParser } from '../../src/shared/application/scripts/parsers.js';
+import { Script } from '../../src/shared/application/scripts/script.js';
+import { ScriptRunner } from '../../src/shared/application/scripts/script-runner.js';
+
+const columnSchemas = [
+  { name: 'challengeId', schema: Joi.string().required() },
+  { name: 'complementaryCertificationKey', schema: Joi.string().required() },
+  { name: 'alpha', schema: Joi.number().required() },
+  { name: 'delta', schema: Joi.number().required() },
+  { name: 'calibrationId', schema: Joi.number().required() },
+];
+
+export class BatchUpdateCertificationFrameworksChallengesFromCsv extends Script {
+  constructor() {
+    super({
+      description: 'update certification-frameworks-challenges alpha and deltas columns from CSV file',
+      permanent: false,
+      options: {
+        file: {
+          type: 'string',
+          describe:
+            'CSV File with challengeId, complementaryCertificationKey, alpha (discriminant), delta (difficulty), and calibrationId columns',
+          demandOption: true,
+          coerce: csvFileParser(columnSchemas),
+        },
+        dryRun: {
+          type: 'boolean',
+          describe: 'Run the script without making any database changes',
+          default: false,
```

Comment:
Protegeons par defaut les executions sauvages, que ca dry run pas de maniere explicite

```suggestion
          default: true,
```